pkgs/development/python-modules/grpc-stubs/default.nix

```
@@ -0,0 +1,50 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  grpcio,
+  pythonOlder,
+  pytestCheckHook,
+  pytest-mypy-plugins,
+  types-protobuf,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "grpc-stubs";
+  version = "1.53.0.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "shabbyrobe";
+    repo = "grpc-stubs";
+    rev = version;
```

My bad, was supposed to be `tag` (which replace `rev` when a tag exists)
```suggestion
    tag = version;
```